napalm_logs/utils/__init__.py

```diff
@@ -174,6 +174,38 @@ def stop(self):
         self.ssl_skt.close()
 
 
+class Replace:
```

Comment:
I'd suggest moving them in a separate module, say `utils/cast.py` or `utils/replace.py` and avoid having the prefix rule.